pkgs/development/tools/documentation/gtk-doc/default.nix

```diff
@@ -14,16 +14,18 @@
 
 python3.pkgs.buildPythonApplication rec {
   pname = "gtk-doc";
-  version = "1.32";
+  version = "1.32.1";
 
+  # TODO: Is this used?
```

Comment:
Yes, this is needed whenever we want to use other builder than the default python one (Meson in this case). Or at least it was needed in the past, if PEP 517 was implemented, it might not be necessary any more.

Reply:
I believe, last time I helped someone package an app like this, it needed `format = "other"`